conda_recipe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set pypi_name = "marshmallow_dataclass" %}
+{% set conda_name = "marshmallow-dataclass" %}
+{% set version = "8.0.0" %}
```

Comment:
Does this need to be manually updated on every release ?

Reply:
see my comment below